### PR TITLE
[BUG] documents.ottawa.ca: navigation and search elements not in a logical tab order

### PR DESCRIPTION
**Pull request by DaveBagler**
See https://github.com/CityofOttawa/ottawa_profile/issues/3051

### CODE REVIEW

File: sass/_layout-overrides.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#main {
+  padding-top: 0;
+}
+
+#navigation {
+  position: static;
+  height: 5em;
+}
+
+@media (min-width: 480px) {
```
Comment (walkerjam) [inline sass/_layout-overrides.scss:14]: Is this just a dup of the mobile case?

File: templates/html.tpl.php [hunk 2]
```diff
@@ -44,7 +44,7 @@
 <body class="<?php print $classes; ?>" <?php print $attributes;?>>
   <?php if ($skip_link_text && $skip_link_anchor): ?>
     <p id="skip-link">
-      <a href="#main-content" class="element-invisible element-focusable"><?php print $skip_link_text; ?></a>
+      <a href="#" onclick="document.getElementById('base-srch').focus({preventScroll:false});" class="element-invisible element-focusable"><?php print t('Skip to main search.'); ?></a>
```
Comment (walkerjam) [inline templates/html.tpl.php:47]: Does this work in IE?

**Review by walkerjam — COMMENTED** [review]
Nice, just a couple little comments/questions.

File: sass/_layout-overrides.scss (hunk 1 above)
Comment (DaveBagler) [reply sass/_layout-overrides.scss:14]: @walkerjam, that's correct.

File: templates/html.tpl.php (hunk 2 above)
Comment (DaveBagler) [reply templates/html.tpl.php:47]: @walkerjam, I don't have a PC with IE set up to test this on, but I did check MDN and caniuse.com, and it looks like the plan `.focus()` method works at least as early as IE 9 and that the scroll toggle works in 10/11. I also did a quick browser stack test (they added it to caniuse.com), and it worked in their IE 11.

**Review by walkerjam — APPROVED** [review]
Nice!

**Pull request merged**